Protos/V1/applications_common.proto

```diff
@@ -55,23 +33,7 @@ message ListApplicationsRequest {
   int32 page = 1; /** The page number. Start at 0. */
   int32 page_size = 2; /** Number of items per page. */
 
-  /**
-   * Represents a filter with all available fields.
-   *
-   * Any fields can be used at the same time.
-   */
-  message Filter {
-    string name = 1; /** Application name. */
-    string version = 2; /** Application version. */
-    string namespace = 3; /** Application namespace. */
-    string service = 4; /** Application service. */
-  }
-
-  /** The filter.
-   *
-   * Must be set for every request but allowed to be empty.
-   */
-  Filter filter = 3;
+  Filters filters = 3; /** The filters. */
```

Comment:
Why 3?

Reply:
because page_size is 2

Reply:
Ah, one filter for each item of the page and one for both?

Reply:
no, it represents the position of the message in grpc serialization process. These numbers must be unique